web/templates/alerts.html

```diff
@@ -9,19 +9,20 @@
     <table class="table table-striped table-bordered table-hover">
       <thead>
         <tr>
+          <th class="alert_expand_col"></th>
           <th>Alert</th>
           <th>Labels</th>
+          <th>Summary</th>
```

Comment:
I'd expect that most of the key information in the summary is already covered by the alert name and labels.

For when someone is quickly scanning the table you want the representation to be as compact as possible.